pantheon-decoupled.php

```diff
@@ -44,3 +45,54 @@ function pantheon_decoupled_graphql_smart_object_cache() {
 }
 
 add_action('init', 'pantheon_decoupled_enable_deps');
+
+function decoupled_settings_init() { 
+
+	register_setting( 'wp-pantheon-decoupled', 'decoupled_settings' );
+
+	add_options_page( 'Pantheon Front-End Sites', 'Pantheon Front-End Sites', 'manage_options', 'wp-pantheon-decoupled', 'decoupled_options_page' );
+
+}
+
+function decoupled_options_page() {
+	?>
+		<div class="wrap">
+			<h1><?php esc_html_e( 'Pantheon Front-End Sites', 'wp-pantheon-decoupled' ); ?></h1>
```

Comment:
My lack of WP knowledge is showing, but I would have expected these escape functions to only be necessary if we were taking dynamic input of some kind. The WP docs you referenced use it like this though, so I'm fine going with this.